operations/firmware/FAS_DBCleanup.md

```diff
@@ -0,0 +1,88 @@
+# Cleaning up FAS Database
+
+FAS stores actions along with all of its operations and snapshots in the database.
+To clean these actions, operations, and snapshots use the following scripts.
```

Comment:
Should this:

> To clean these actions, operations, and snapshots use the following scripts.


Instead read:

> To remove actions and snapshots, use the following scripts.


Since the only two scripts remove actions and snapshots, not operations?

Reply:
Operations are removed with actions, but it does make it confusing, so updated.